lib/master.js

```diff
@@ -195,9 +195,35 @@ Master.prototype._startWorkers = function(remainingWorkers) {
                 type: 'config',
                 body: yaml.dump(self.config)
             });
+
+            var workerExit = function(code) {
```

Comment:
Hm, this procedure breaks the sequential fork of workers. Instead of letting the normal worker restart procedure handle that, wouldn't it be better simply not to decrease `remainingWorkers` ?